pkg/resources/management.cattle.io/v3/project/quota_validate.go

```diff
@@ -0,0 +1,43 @@
+package project
+
+import (
+	mgmtv3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/wrangler/pkg/data/convert"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	quotav1 "k8s.io/apiserver/pkg/quota/v1"
+)
+
+// quotaFits checks whether the quota in the second argument is sufficient for the requested quota in the first argument.
+// If it is not sufficient, a list of the resources that exceed the allotment is returned.
+// The ResourceList to be checked can be compiled by passing a
+// ResourceQuotaLimit to convertLimitToResourceList before calling this
+// function on the result.
+func quotaFits(resourceListA corev1.ResourceList, resourceListB corev1.ResourceList) (bool, corev1.ResourceList, error) {
```

Comment:
And you don't need to return an error from here anymore, since it's always nil.

Reply:
fixed